model.js

```diff
@@ -0,0 +1,13 @@
+const db = require('./db/connection')
+
+async function selectAllTopics(){
+  try{
+    const query = await db.query(`SELECT * FROM topics`)
+    return query
+  }
+  catch(err){
+    throw err
+  }
```

Comment:
We don't really need to have a catch here as our controller will catch an error and pass it onto our error handlers.